pymess/management/commands/sync_emails.py

```diff
@@ -20,8 +20,14 @@ def _sync_for_locale(self, base_directory, locale):
             filename for filename in os.listdir(locale_directory) if filename.endswith('.html')
         ]
         for filename in files_with_email_html_body:
+            if filename.endswith('_B.html'):
+                slug = filename[:-7]  
+                variant = 'B'
+            else:
+                slug = filename[:-5]  
+                variant = 'A'  
```

Comment:
This is not a good solution. Variants are up to the library users to choose from.

Make `variant` optional input for the command and use it to filter all files in expression above:

```python
        files_with_email_html_body = [
            filename for filename in os.listdir(locale_directory) if filename.endswith(f'{variant}.html')
        ]
```